modules/rubiconAnalyticsAdapter.js

```diff
@@ -479,7 +608,7 @@ let rubiconAdapter = Object.assign({}, baseAdapter, {
             delete bid.error; // it's possible for this to be set by a previous timeout
             break;
           case NO_BID:
-            bid.status = args.status === BID_REJECTED ? 'rejected' : 'no-bid';
+            bid.status = args.status === BID_REJECTED ? 'rejected-ipf' : 'no-bid';
```

Comment:
why `-ipf`?  Is this the only reason a bid could be rejected?  Maybe we use a status called `BID_REJECTED_IPF` to make this more explicit?

Reply:
There will be future rejected reasons, so updating with -ipf now.

But I'll make it a constant.